Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG RUBY_IMAGE
+FROM ${RUBY_IMAGE}
+
+ARG VENDOR_PATH
+ARG BUNDLER_VERSION
+
+# For tzdata
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN [ apt-get ] \
```

Comment:
This conditional might not be required if the RUBY_IMAGE is Debian based, or are any plans to support other OS flavors?

Reply:
True! I copied it from the other Dockerfile where we also have support for `yum`.